addons/weaponCfgFixes/CfgWeapons.hpp

```diff
@@ -0,0 +1,14354 @@
+class Mode_FullAuto;
+class Mode_Burst;
+class Mode_SemiAuto;
+class CfgWeapons {
```

Comment:
It might look strange in SQF, but I think we can cut this down a good bit by using defines

Reply:
Can you elaborate on what you mean by "using defines"?